scripts/create-clusters.sh

```diff
@@ -155,7 +156,10 @@ function check-clusters-ready() {
 }
 
 function configure-insecure-registry-on-cluster() {
-  configure-insecure-registry-and-reload "docker exec cluster${1}-control-plane bash -c" '$(pgrep dockerd)'
+  cmd_context="docker exec cluster${1}-control-plane bash -c "
+  containerd_id=`${cmd_context} "pgrep -x containerd"`
+  ${cmd_context} "sed -i '/\[plugins.cri.registry.mirrors\]/a [plugins.cri.registry.mirrors."\"${CONTAINER_REGISTRY_HOST}\""]\nendpoint = ["\"http://${CONTAINER_REGISTRY_HOST}\""]' ${containerd_config}"
```

Comment:
Consider adding this config command into a new function similar to `insecure-registry-config-cmd`. Then add a 3rd parameter to `configure-insecure-registry-and-reload` that contains the name of the function to invoke that provides the command to configure the insecure registry. The result is that `configure-insecure-registry-and-reload` can then continue to be called by this function with the 3rd argument being the name of this new function. This allows `configure-insecure-registry-and-reload` to continue to be called for host or cluster and for dockerd or containerd.